roles/traefik/templates/traefik.toml.j2

```diff
@@ -13,3 +13,14 @@ domain = "{{ traefik_marathon_domain }}"
 watch = true
 networkInterface = "{{ ansible_default_ipv4['interface'] }}"
 ExposedByDefault = {{ traefik_marathon_expose_by_default }}
+
+defaultEntryPoints = ["http", "https"]
+[entryPoints]
+  [entryPoints.http]
+  address = ":80"
+  [entryPoints.https]
+  address = ":443"
+    [entryPoints.https.tls]
+      [[entryPoints.https.tls.certificates]]
+      CertFile = "/etc/traefik/certs/traefik-admin.cert.pem"
+      KeyFile = "/etc/traefik/certs/traefik-admin.key.pem"
```

Comment:
That's been removed from master: https://github.com/CiscoCloud/mantl/commit/c73b294f673314a84d656d07012620937d487f21